File: app/src/main/java/com/example/mvctutorial/designpattern/_09bidge/SoundMCF.java
package com.example.mvctutorial.designpattern._09bidge;

public class SoundMCF implements MorseCodeFunction{
    @Override
    public void dot() {
        System.out.print("bip");
    }

    @Override
    public void dash() {
        System.out.print("bip~");
    }

    @Override
    public void space() {
        System.out.print(" ");
    }
}

File: app/src/main/java/com/example/mvctutorial/designpattern/_09bidge/SoundMCF.kt
package com.example.mvctutorial.designpattern._09bidge

class SoundMCF: MorseCodeFunction {
    override fun dot() {
        println("b")
    }

    override fun dash() {
        println("B")
    }

    override fun space() {
        println(" ")
    }
}